code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -341,6 +416,17 @@
 	if(C.getToxLoss() >= heal_threshold)
 		return TRUE
 
+/mob/living/simple_animal/bot/medbot/attack_hand(mob/living/carbon/human/H)
+	if(H.a_intent == INTENT_DISARM && mode != BOT_TIPPED)
+		H.visible_message("<span class='danger'>[H] begins tipping over [src].</span>", "<span class='danger'>You begin tipping over [src]...</span>")
+		speak(pick(list("Hey, wait...", "Please don't...", "Whoa, hold on-", "Wait!-", "I trusted you!")))
+		if(do_after(H, 3 SECONDS, target=src))

```

Comment:
this do_after makes it kind of hard to tip over a medibot that's on patrol, but at the same time, being able to instantly tip over pAI medibots would be kind of lame on the pAI medibot's end.

maybe remove this do_after() check for NPC medibots, but make medibots that have minds require the do_after to be tipped over?

actually, how does this whole medibot panic system interact with player-controlled medibots? can they ever right themselves? does being tipped over even AFFECT a player-controlled medibot, or can you just roam around as an Australian medibot?